lib/presentation/bill_details/bill_details_page.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:gap/gap.dart';
+import 'package:material_symbols_icons/symbols.dart';
+
+import '../../utils/styles/app_color.dart';
+import 'components/bottom_modal.dart';
+import 'components/grade_based_features.dart';
+import 'components/plan_table.dart';
+import 'components/premium_plan.dart';
+
+class BillDetailsPage extends StatelessWidget {
+  const BillDetailsPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    const imagePath = 'assets/images/billing_header.png';
+
+    const bottomModalHeight = 150.0;
+
+    return Scaffold(
+      body: SafeArea(
+        top: false,
+        child: Padding(
+          padding: const EdgeInsets.only(bottom: bottomModalHeight),
+          child: CustomScrollView(
+            slivers: [
+              SliverAppBar(
+                systemOverlayStyle: const SystemUiOverlayStyle(
+                  statusBarBrightness: Brightness.light,
+                ),
+                expandedHeight: 250,
+                flexibleSpace: FlexibleSpaceBar(
+                  background: Image.asset(imagePath, fit: BoxFit.cover),
```

Comment:
せっかく`assets.gen.dart`ファイルを生成してるので、文字列をハードコードせず、安全に呼び出してあげましょう！

```suggestion
                  background: Image.asset(Assets.images.billingHeader, fit: BoxFit.cover),
```

Reply:
使うの忘れてた...ありがとうDone！